test/CMakeLists.txt

```diff
@@ -267,22 +262,16 @@ if(PYTHON_BINDINGS)
   )
 
   foreach(pybindtest ${pybindtests})
-    SET_SOURCE_FILES_PROPERTIES(test${pybindtest}.py PROPERTIES
-      PYTHONPATH "${CMAKE_SOURCE_DIR}/scripts/python:${CMAKE_BINARY_DIR}/lib${LIB_SUFFIX}"
-      BABEL_LIBDIR "${FORMATDIR}"
-      BABEL_DATADIR "${CMAKE_SOURCE_DIR}/data"
-      LD_LIBRARY_PATH "${CMAKE_BINARY_DIR}/lib${LIB_SUFFIX}:$ENV{LD_LIBRARY_PATH}"
-    )
-
-    if(MSVC)
-      SET_SOURCE_FILES_PROPERTIES(test${pybindtest}.py PROPERTIES
-        PYTHONPATH "\"${CMAKE_SOURCE_DIR}/scripts/python;${CMAKE_BINARY_DIR}/bin/Release\""
```

Comment:
I don’t know Windows environment variable functionality in detail, but it seems wrong.